ProjectTemplates/templates/Worker-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,188 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common",
+    "Worker",
+    "Web"
+  ],
+  "name": "Worker Service",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An empty project template for creating a worker service.",
```

Comment:
An empty project template for creating a worker service which supports authentication with Azure AD and Azure AD B2C  ?